tutorials/image/cifar10_estimator/cifar10_utils.py

```diff
@@ -0,0 +1,137 @@
+import six
+
```

Comment:
need to add.  Not sure why it did not error on you.  
import collections  
import tensorflow as tf  <-  due to tf.contrib.learn.RunConfig